slidechain/export.go

```diff
@@ -166,9 +166,9 @@ func (c *Custodian) pegOutFromExports(ctx context.Context, pegouts chan<- pegOut
 			if err != nil {
 				peggedOut = pegOutFail
 				if herr, ok := errors.Root(err).(*horizon.Error); ok {
-					resultCodes, err := herr.ResultCodes()
-					if err != nil {
-						log.Fatalf("getting error codes from failed submission of tx %x", txid)
+					resultCodes, rerr := herr.ResultCodes()
+					if rerr != nil {
+						log.Fatalf("getting error codes from failed submission of tx %x (with horizon err '%s' from original error '%s'): %s", txid, herr, err, rerr)
```

Comment:
My apologies: on further reflection, including both `herr` and `err` here is redundant, because `herr` _is_ `err`.